### PR TITLE
controller layer is added; userDto created; get by id method added 

### PR DESCRIPTION
**Pull request by andromash**
… user

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,19 +17,50 @@
...
+            <groupId>javax.servlet.jsp</groupId>
+            <artifactId>jsp-api</artifactId>
+            <version>2.1.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>jstl</artifactId>
+            <version>1.2</version>
+        </dependency>
```
Comment (Rommelua) [inline pom.xml]: We don't need these two dependecies

File: src/main/java/spring/intro/config/WebConfig.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
+        InternalResourceViewResolver resolver = new InternalResourceViewResolver();
+        resolver.setViewClass(JstlView.class);
+        resolver.setPrefix("/WEB-INF/views/");
+        resolver.setSuffix(".jsp");
+        return resolver;
+    }
```
Comment (Rommelua) [inline src/main/java/spring/intro/config/WebConfig.java]: we don't need this method

File: src/main/java/spring/intro/dao/UserDaoImpl.java [hunk 3]
```diff
@@ -45,4 +45,14 @@ public User add(User user) {
             return getAllUsers.getResultList();
         }
     }
+
+    @Override
+    public User get(Long userId) {
+        try (Session session = sessionFactory.openSession()) {
+            return session.createQuery("from User where id = :id", User.class)
+                    .setParameter("id", userId)
+                    .getSingleResult();
```
Comment (Rommelua) [inline src/main/java/spring/intro/dao/UserDaoImpl.java]:
```suggestion
            return session.get(User.class, id);
```

File: pom.xml (hunk 1 above)
Comment (andromash) [reply pom.xml]: agree, forgot to delete

File: src/main/java/spring/intro/dao/UserDaoImpl.java (hunk 3 above)
Comment (andromash) [reply src/main/java/spring/intro/dao/UserDaoImpl.java]: wow, that's so simple, thanks :)

**Review by Rommelua — APPROVED** [review]

File: src/main/java/spring/intro/config/WebConfig.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"spring.intro.controllers"})
+public class WebConfig {
+
```
Comment (RomanDubovskyi) [inline src/main/java/spring/intro/config/WebConfig.java:11]: redundant empty line

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**